pkg/steps/index_generator_test.go

```diff
@@ -65,16 +67,34 @@ WORKDIR /index-data
 COPY --from=builder index.Dockerfile index.Dockerfile
 COPY --from=builder /database/ database`,
 	}, {
-		name: "multiple bundles",
+		name: "single bundle, no pull secret",
```

Comment:
The `diff` here is not great.  All existing tests are unchanged except for the extra `pullSecret` field in each `step`.

This new test is the same as the previous, but omits the secret.

Reply:
Could we add a new case instead of modifying the existing one?

Reply:
Not entirely since the new implementation alters the content generated for the existing tests, but I've added a duplicate of each previous test which has a pull secret.